src/Application.vala

```diff
@@ -17,13 +17,13 @@
 public class AppCenter.App : Granite.Application {
     public const OptionEntry[] APPCENTER_OPTIONS =  {
         { "show-updates", 'u', 0, OptionArg.NONE, out show_updates,
-        "Display the Installed Panel", null},
+        N_("Display the Installed Panel"), null},
         { "silent", 's', 0, OptionArg.NONE, out silent,
-        "Run the Application in background", null},
+        N_("Run the Application in background"), null},
```

Comment:
Mixed style - use definite article consistently in accordance with elementary style.